test/libsolidity/syntaxTests/nameAndTypeResolution/463_error_transfer_non_payable_fallback.sol

```diff
@@ -9,9 +9,7 @@ contract B {
     A a;
 
     function() external {
-        a.transfer(100);
+        address(a).transfer(100);
     }
 }
 // ----
-// Warning: (213-223): Using contract member "transfer" inherited from the address type is deprecated. Convert the contract to "address" type to access the member, for example use "address(contract).transfer" instead.
-// TypeError: (213-223): Value transfer to a contract without a payable fallback function.
```

Comment:
Since this test case is not failing anymore, perhaps the file should be renamed, if not just outright removed (does it test anything meaningful?)

Reply:
Yea I think we should remove them